### PR TITLE
Update has transformed

### PR DESCRIPTION
**Pull request by kameyer226**
added native_limits as a config option for has-transformed-position trait

### CODE REVIEW

**Comment by ddkohler** [comment]
@kameyer226 I got a message to review this PR, but it looks like it's not done yet?  I'm expecting this PR will resolve https://github.com/yaq-project/yaq-python/issues/56 . Let me know if there is something you want checked at this point.

**Comment by ddkohler** [comment]
> @kameyer226 I got a message to review this PR, but it looks like it's not done yet? I'm expecting this PR will resolve [yaq-project/yaq-python#56](https://github.com/yaq-project/yaq-python/issues/56) . Let me know if there is something you want checked at this point.

My bad--I didn't check the repo and thought that this was a PR for yaq-python...

File: yaq_traits/has-transformed-position.toml [hunk 1]
```diff
@@ -8,6 +8,15 @@ of low-level motor control.
 requires = ["has-limits"]
 
 
+[config]
+
+[config.native_limits]
+type = "array"
+items = "double"
+default = [-inf, +inf]
+doc = "Configuration native_limits are strictly optional."
```
Comment (ddkohler) [inline yaq_traits/has-transformed-position.toml:17]:
```suggestion
doc = "Bounds of motion specified in native coordinates.  To specify bounds in transformed coordinates, use `limits`."
```

**Review by ddkohler — APPROVED** [review]
Aside from a better docstring, looks good!

**Comment by untzag** [comment]
@kameyer226 @ddkohler please update the YEP.

**Pull request merged**